src/literature/components/NumberOfPages.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class NumberOfPages extends Component {
+  render() {
+    const { numberOfPages } = this.props;
+
+    if (!numberOfPages) {
```

Comment:
this won't render, also when this value is `0`, is it what we want? @michamos

Reply:
it's strictly positive according to the schema https://github.com/inspirehep/inspire-schemas/blob/5d129dcdb5dda1584381c9c253cc7d1b8a499e0f/inspire_schemas/records/hep.yml#L1173-L1176, so it should be displayed if `number_of_pages` is present.